core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2823,73 +2914,313 @@ public void cleanUp() {
                 public void execute(Runnable command) {
                     command.run();
                 }
+            }, new ReactorListener() {
+                final Level level = Level.parse(Configuration.getStringConfigParameter("termLogLevel", "FINE"));
```

Comment:
:ant: Why should this be a configuration option? Log at `FINE` and if the user wishes to see otherwise hidden messages, they can create a custom logger for that purpose.